middleware/redirects.ts

```diff
@@ -1,27 +1,49 @@
 export default function ({ redirect, route }): void {
   const { urlPrefix } = usePrefix()
 
-  if (route.path === '/drops') {
-    return redirect('/ahk/drops')
-  }
+  let redirectValue
 
-  if (route.path.startsWith(`/${urlPrefix.value}`)) {
-    if (route.path.endsWith('collections')) {
-      return redirect(`/${urlPrefix.value}/explore/collectibles`)
-    }
-    if (route.path.endsWith('gallery')) {
-      return redirect(`/${urlPrefix.value}/explore/items`)
-    }
-  }
+  const paths = [
+    {
+      cond: (val) => val === '/drops',
+      replaceValue: '/ahk/drops',
+    },
+    {
+      replaceValue: () => `/${urlPrefix.value}/explore/collectibles`,
+      cond: (val) =>
+        val.startsWith(`/${urlPrefix.value}`) && val.endsWith('collections'),
+    },
+    {
+      cond: (val) =>
+        val.startsWith(`/${urlPrefix.value}`) && val.endsWith('gallery'),
+      replaceValue: () => `/${urlPrefix.value}/explore/items`,
+    },
+    {
+      cond: (val) => val.includes('/stmn/'),
+      replaceValue: () => window.location.href.replace('/stmn/', '/ahk/'),
+    },
+    {
+      cond: (val) => val.includes('/rmrk2/'),
+      replaceValue: () => window.location.href.replace('/rmrk2/', '/ksm/'),
+    },
+    {
+      cond: (val) => val.startsWith('/transfer'),
+      replaceValue: () =>
+        window.location.href.replace('/transfer', '/ksm/transfer'),
+    },
+  ]
 
-  if (route.path.includes('/stmn/')) {
-    return redirect(window.location.href.replace('/stmn/', '/ahk/'))
+  for (const path of paths) {
+    if (path.cond(route.path)) {
+      redirectValue =
+        typeof path.replaceValue === 'function'
+          ? path.replaceValue()
+          : path.replaceValue
+      break
+    }
   }
 
-  if (route.path.includes('/rmrk2/')) {
-    return redirect(window.location.href.replace('/rmrk2/', '/ksm/'))
-  }
-  if (route.path.startsWith('/transfer')) {
-    return redirect(window.location.href.replace('/transfer', '/ksm/transfer'))
+  if (redirect) {
```

Comment:
This condition will always be true.

```suggestion
  if (redirectValue) {
```